test_helpers/main.go

```diff
@@ -185,6 +187,13 @@ func StartTarantool(startOpts StartOpts) (TarantoolInstance, error) {
 		return inst, err
 	}
 
+	netOpts := strings.Split(startOpts.Listen, ":")
+	ln, err := net.Listen("tcp", ":"+netOpts[1])
```

Comment:
Because of issue described in commit message.

Reply:
> Because of issue described in commit message.

Is there any advantage to listen ":host" instead of just "host:port"? I understand a description from [net.Listen](https://pkg.go.dev/net#Listen):

```
The address can use a host name, but this is not recommended, because it will create a listener for at most one of the host's IP addresses.
```

But I don't think it matters in this case.